src/theme/css/noscript.css

```diff
@@ -0,0 +1,7 @@
+.left-buttons button {
+    display: none;
+}
+
+.sidebar {
+    transition: transform 0.3s; /* Animation: slide away */
+}
```

Comment:
The normal CSS file has this in it:

```css
body:not(.sidebar-resizing) .sidebar {
    transition: transform 0.3s; /* Animation: slide away */
}
```

Is there some reason why this doesn't work?

Reply:
I think this one can be removed indeed.